File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.java
class Solution {
    public String removeVowels(String s) {
       HashSet<Character> vowels = new HashSet<>();
       vowels.add('a');
       vowels.add('e');
       vowels.add('i');
       vowels.add('o');
       vowels.add('u');

        StringBuilder sb  = new StringBuilder();

        for(char c : s.toCharArray()){
            if(!vowels.contains(c)){
                sb.append(c);
            }
        }

        return sb.toString();
    }
}

File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.py
class Solution:
    def removeVowels(self, s: str) -> str:
        vowels = {'a','e','i','o', 'u'}
        ans = []
        for letter in s:
            if letter not in 'aeiou':
                ans.append(letter)
        return ''.join(ans)